RubberduckTests/Inspections/AssignedByValParameterInspectionTests.cs

```diff
@@ -97,19 +97,152 @@ public void AssignedByValParameter_ReturnsResult_SomeAssignedByValParams()
         [TestCategory("Inspections")]
         public void AssignedByValParameter_QuickFixWorks()
```

Comment:
Add a bracketed parameter test which failed - the ArgContext name is now [bar].  Of course, it  can be made to pass by inserting an OR statement to accept a bracketed version of the identifier.  That said, I tried creating a Sub in Excel with a bracketed param...and it would not allow it.  Is this construct something allowed in Access or past versions of Excel?